NitroxServer/GameLogic/TimeKeeper.cs

```diff
@@ -61,12 +65,15 @@ public double ElapsedSeconds
     /// </remarks>
     private const int RESYNC_INTERVAL = 60;
 
-    public TimeKeeper(PlayerManager playerManager, double elapsedSeconds)
+    public TimeSkipped OnTimeSkipped;
```

Comment:
`On..` prefix is usually used for event handler methods, not as event name.

```suggestion
    public TimeSkipped TimeSkipped;
```